dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -122,7 +122,7 @@ The file Makefile.PL should look something like this:
 	VERSION_FROM => 'Mytest.pm', # finds $VERSION
 	LIBS         => [''],   # e.g., '-lm'
 	DEFINE       => '',     # e.g., '-DHAVE_SOMETHING'
-	INC          => '',     # e.g., '-I/usr/include/other'
+	INC          => '-I.', # e.g., '-I. -I/usr/include/other'
```

Comment:
By no means should `-I.` be recommended to users.

Reply:
It's in the `h2xs -A -n Mytest` output Makefile.PL.

Reply:
Did you test run h2xs in a directory with a `*.h` file in it?

h2xs helpfully adds the `-I.` if it finds any `*.h` or `*.hh` files and I don't think that's the case if you follow the steps in perlxsitut.

Reply:
https://perldoc.perl.org/perlxstut

EXAMPLE 1

Run `"h2xs -A -n Mytest"`.

```
bash-4.3$ ls /tmp/*.h
ls: cannot access '/tmp/*.h': No such file or directory
bash-4.3$ ls /tmp/*.hh
ls: cannot access '/tmp/*.hh': No such file or directory
bash-4.3$ cd /tmp && h2xs -A -n Mytest
Defaulting to backwards compatibility with perl 5.22.2
If you intend this module to be compatible with earlier perl versions, please
specify a minimum perl version with the -b option.

Writing Mytest/ppport.h
Writing Mytest/lib/Mytest.pm
Writing Mytest/Mytest.xs
Writing Mytest/Makefile.PL
Writing Mytest/README
Writing Mytest/t/Mytest.t
Writing Mytest/Changes
Writing Mytest/MANIFEST
bash-4.3$ grep -nr other Mytest/Makefile.PL
16:    INC               => '-I.', # e.g., '-I. -I/usr/include/other'
bash-4.3$ rm -rf Mytest
bash-4.3$ 
```